web_pwa_oca/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+To configure your PWA:
+
+* Go to **Settings > General Settings > Progressive Web App**.
+* Set the parameters (*Note:* Icon **must be a SVG file**)
+* **Save**
```

Comment:
If you're in USAGE, you shouldn't explain how to configure. That belongs to CONFIGURE.